### PR TITLE
Added bin files for frontend and server packages

### PR DESCRIPTION
**Pull request by PatrickShaw**
This is another step toward releasable code: People can install the frontend and backend packages and actually run them (or build a bundle, in the case of the frontend).

Had to refactor the mocking and logger logic in order to get tests working properly for this PR.

The actual browser code is now in a package called frontend-dist. This allows others to use their own bundling tools.

Also noticed the logger wasn't able to log exceptions so I fixed that. Sorry for the multi-feature PR.

Any extra features, like a `--help` or configurable options can be added in another PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,20 @@ Project in development - Open sourced service overlay for SDK management using s
...
+```
+yarn add global @openapi-platform/frontend @openapi-platform/server
+```
+
+## How to use it
+1. Run `start-openapi-platform-server`
+2. Run `start-openapi-platform-frontend`
+
+## Configuration
+You can create a config file called `openapi-platform.config.xxx` where xxx is a `json`, `yaml` or `json5` extension.
```
Comment (calebj0seph) [inline README.md]: Specify that this config file needs to be in the current directory when running either of the above commands.

File: packages/frontend/bin/start-openapi-platform-frontend.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+build({
+  OUTPUT_PATH: __dirname,
+})
+  .then(() => {
+    const config = readConfig();
+    /* 
+    Note that if people want to customize this 
+    they can just use build-openapi-platform-frontend
+  */
```
Comment (calebj0seph) [inline packages/frontend/bin/start-openapi-platform-frontend.js]: Can you either make the `*/` the same indentation as `/*` or use `//` style comments?

File: packages/frontend/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import webpack from 'webpack';
+import createWebpackConfig from '../webpack.config';
+
+export async function build({ ...webpackOptions } = {}) {
+  const logger = openapiLogger();
+  logger.info('Bundling frontend app...');
+
+  const openapiConfig = readConfig();
+  const webpackConfig = createWebpackConfig({
+    OUTPUT_PATH: process.cwd(),
```
Comment (calebj0seph) [inline packages/frontend/src/index.ts]: I'm guessing we'll allow this to be configurable later? If so might want to add a TODO.

File: packages/logger/src/index.ts [hunk 4]
```diff
@@ -5,6 +5,34 @@ import { createLogger, format, transports } from 'winston';
 
 const logLevel: string = process.env.LOG_LEVEL || 'debug';
 
+const openapiPlatformErrors = format(info => {
+  if (info instanceof Error) {
+    /*
```
Comment (calebj0seph) [inline packages/logger/src/index.ts]: Again, use consistent indentation for the comments or use `//`.

File: packages/frontend/src/index.ts (hunk 3 above)
Comment (PatrickShaw) [reply packages/frontend/src/index.ts]: Added one in the bin files. Note that there's `...webpackOptions` in that parameter input.

**Pull request merged**